README.md

```diff
@@ -233,17 +225,20 @@ return static function (ContainerConfigurator $containerConfigurator): void {
 For in-file exclusion, use `@noRector \FQN name` annotation:
 
 ```php
+<?php
+declare(strict_types=1);
+
 class SomeClass
 {
     /**
      * @noRector
      */
-    const NAME = '102';
+    public const NAME = '102';
 
     /**
      * @noRector
      */
-    public function foo()
+    public function foo(): void
```

Comment:
here seems on purpose for `@noRector` to exclude from check, so it needs revert.

Reply:
It's actually related to Rector, not ECS. Coding standard is correct here